samples/csharp/getting-started/AnomalyDetection_CreditCardFraudDetection/CreditCardFraudDetection.Predictor/Program.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.IO;
+
+using CreditCardFraudDetection.Common;
+
+namespace CreditCardFraudDetection.Predictor
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            string assetsPath = GetAbsolutePath(@"../../../assets");
+            string trainOutput = GetAbsolutePath(@"../../../../CreditCardFraudDetection.Trainer/assets/output");
+
+            CopyModelAndDatasetFromTrainingProject(trainOutput, assetsPath);
+
+            var inputDatasetForPredictions = Path.Combine(assetsPath, "input", "testData.csv");
+            var modelFilePath = Path.Combine(assetsPath, "input", "randomizedPca.zip");
+
+            // Create model predictor to perform a few predictions
+            var modelPredictor = new Predictor(modelFilePath, inputDatasetForPredictions);
+
+            modelPredictor.RunMultiplePredictions(numberOfPredictions: 5);
+
+            Console.WriteLine("=============== Press any key ===============");
+            Console.ReadKey();
+        }
+
+
+        public static void CopyModelAndDatasetFromTrainingProject(string trainOutput, string assetsPath)
+        {
+            if (!File.Exists(Path.Combine(trainOutput, "testData.csv")) ||
```

Comment:
If testData.csv doesn't exist, wouldn't that be a different error than not running the training project first? Would you want to give a different/separate error for that case?

Reply:
As it stands now, the training project splits the original dataset into train/test.  So both the trained model (`reandomizedPca.zip`) and the test data csv (`testData.csv`) are copied into the `CreditCardFraudDetection.Trainer`s output directory upon running the trainer project.  So as it currently works, it is the same error, as `CreditCardFraudDetection.Predictor` is looking for both files output from the trainer project.

Reply:
Makes sense!